agent/src/agent/pipeline/config/jython_scripts/tests/influx2.py

```diff
@@ -0,0 +1,29 @@
+global sdc
```

Comment:
please rename this `tests` directory to `test_pipelines`

Reply:
Renamed